code/statsd/src/main/scala/com/avast/datadog4s/statsd/event/EventerImpl.scala

```diff
@@ -0,0 +1,45 @@
+package com.avast.datadog4s.statsd.event
+
+import cats.effect.Sync
+import com.avast.datadog4s.api.Tag
+import com.avast.datadog4s.api.event.{ Eventer, _ }
+import com.timgroup.statsd.{ StatsDClient, Event => SEvent }
+
+import scala.collection.immutable.Seq
+class EventerImpl[F[_]: Sync](statsDClient: StatsDClient, defaultTags: Seq[Tag]) extends Eventer[F] {
+  private[this] val F = Sync[F]
+
+  private val toPriority: Priority.Value => SEvent.Priority = {
+    case Priority.Low    => SEvent.Priority.LOW
+    case Priority.Normal => SEvent.Priority.NORMAL
+  }
+
+  private val toAlertType: AlertType.Value => SEvent.AlertType = {
+    case AlertType.Error   => SEvent.AlertType.ERROR
+    case AlertType.Warning => SEvent.AlertType.WARNING
+    case AlertType.Info    => SEvent.AlertType.INFO
+    case AlertType.Success => SEvent.AlertType.SUCCESS
+  }
+
+  override def send(
+    event: Event,
+    tags: Tag*
+  ): F[Unit] =
+    F.delay {
+      statsDClient.recordEvent(
+        SEvent
+          .builder()
+          .withTitle(event.title)
+          .withText(event.text)
+          .withDate(event.date.map(_.toEpochMilli).getOrElse[Long](-1))
```

Comment:
I'M not sure `-1` is correct here.

Reply:
its what the date defaults to in the [source](https://github.com/DataDog/java-dogstatsd-client/blob/master/src/main/java/com/timgroup/statsd/Event.java#L12), although I only did it to get away from `.foreach` but having the default here is not great if the library updates

Reply:
im that case i think `foreach` is more futureproof and probably better solution